tests/ert_tests/gui/models/test_job_list_model.py

```diff
@@ -70,28 +73,65 @@ def test_changes(full_snapshot):
         ),
     )
     source_model._add_partial_snapshot(SnapshotModel.prerender(partial), 0)
-    assert model.index(0, _id_to_col(ids.START_TIME), QModelIndex()).data() == str(
-        start_time
-    )
-    assert model.index(0, _id_to_col(ids.END_TIME), QModelIndex()).data() == str(
-        end_time
+    assert (
+        model.index(0, _id_to_col(ids.DURATION), QModelIndex()).data()
+        == "1 day, 1:00:00"
     )
     assert (
         model.index(0, _id_to_col(ids.STATUS), QModelIndex()).data()
         == JOB_STATE_FAILURE
     )
 
 
-def test_no_cross_talk(full_snapshot):
+@pytest.mark.parametrize("timezone", [(None), (tz.gettz("UTC"))])
+@patch("ert_gui.model.snapshot.datetime", wraps=datetime)
+def test_duration(mock_datetime, timezone, full_snapshot):
     source_model = SnapshotModel()
 
     model = JobListProxyModel(None, 0, 0, 0, 0)
     model.setSourceModel(source_model)
 
     reporting_mode = qt_api.QtTest.QAbstractItemModelTester.FailureReportingMode.Warning
-    tester = qt_api.QtTest.QAbstractItemModelTester(  # noqa, prevent GC
-        model, reporting_mode
+    qt_api.QtTest.QAbstractItemModelTester(model, reporting_mode)  # noqa, prevent GC
+
+    source_model._add_snapshot(SnapshotModel.prerender(full_snapshot), 0)
+    assert (
+        model.index(0, _id_to_col(ids.STATUS), QModelIndex()).data() == JOB_STATE_START
+    )
+
+    partial = PartialSnapshot(full_snapshot)
+    start_time = datetime.datetime(
+        year=2020, month=10, day=27, hour=12, tzinfo=timezone
+    )
+    # mock only datetime.datetime.now()
+    mock_datetime.datetime.now.return_value = datetime.datetime(
+        year=2020, month=10, day=28, hour=13, tzinfo=timezone
+    )
```

Comment:
Maybe include microseconds to test/document that behavior as well?

Reply:
Yepp, done!